src/SFA.DAS.Reservations.Web/Controllers/HomeController.cs

```diff
@@ -29,12 +30,22 @@ public IActionResult SignOut(string ukprn="", string employerAccountId="")
                 CookieAuthenticationDefaults.AuthenticationScheme,
                 WsFederationDefaults.AuthenticationScheme);
         }
-
+        
         [Route("notAvailable", Name="FeatureNotAvailable")]
         public IActionResult FeatureNotAvailable()
         {
             return View();
         }
-         
+
+        [Route("{employerAccountId}/service/password/change", Name =RouteNames.EmployerChangePassword)]
+        public IActionResult ChangePassword(ReservationsRouteModel model,bool userCancelled = false)
+        {
+            return RedirectToRoute(RouteNames.EmployerIndex,model);
```

Comment:
how do they change the password if just redirect back to index? (same for ChangeEmail below)

Reply:
The change happens within employer users, as part of the change pwd, email request you need to specify the returnurl which needs to match the url defined in the relying party. The bit missing is a banner message saying success